### PR TITLE
#781 Excude inactice users from notifications

### PR DESCRIPTION
**Pull request by KlemenSpruk**

### CODE REVIEW

File: vue/components/notifications-editor.vue [hunk 1]
```diff
@@ -14,19 +14,50 @@ import {
...
+    type: String,
+    required: false,
+  },
+  licenseConsumerUrlTrailingSlash:{
+    type: Boolean,
+    required: false,
+  },
+}>();
+
+const consumerUrl: string = (props.consumerUrl !== undefined ? props.consumerUrl : 'notification') as string;
```
Comment (velis74) [inline vue/components/notifications-editor.vue:36]: Za tole imaš DetailViewApi, sploh še s pull requestom, ki dodaja še search parametre.

**Review by velis74 — CHANGES_REQUESTED** [review]
unit tests, docs

File: vue/components/notifications-editor.vue (hunk 1 above)
Comment (KlemenSpruk) [reply vue/components/notifications-editor.vue:36]: tukaj ne rabim detail api view

**Comment by KlemenSpruk** [comment]
Plugin za coverage ne dela. 

Run coverage run -m unittest discover tests/
EEEEEEEEEE
======================================================================

File: vue/components/notifications-editor.vue (hunk 1 above)
Comment (velis74) [reply vue/components/notifications-editor.vue:36]: Sem samo predlagal, ker si kuril 20 vrstic kode, namesto treh.
Coverage check ti odleti, doddaj kak unit test.

**Comment by KlemenSpruk** [comment]
pr popravljen

**Review by velis74 — APPROVED** [review]

**Pull request merged**